lib/tic_tac_toe.rb

```diff
@@ -1,7 +1 @@
-class Game
-    def what_is_the_name_of_the_game?
-      "Welcome to Tic-Tac-Toe!"
-    end
-end
-
-puts "Welcome to Tic-Tac-Toe!"
\ No newline at end of file
+puts "Welcome to Tic Tac Toe!"
```

Comment:
I understand that this print statement was put here for the walking skeleton. We probably don't need this file anymore though!